.github/workflows/python-build-and-test.yml

```diff
@@ -2,20 +2,88 @@ name: Python unit tests
 
 on: [push]
 
+env:
+  PYTHONPATH: ${{ github.workspace }}
+
 jobs:
-  build:
+  build1:
```

Comment:
try more explicit naming, this one is running integration test and EVM test so it's probably `integration-and-evm-test`